### PR TITLE
Reduce delay from 5 to 1 second, remove initial operation call, remov…

### PR DESCRIPTION
**Pull request by drwill-ms**
…e checking after close of device client, and remove wait after

### CODE REVIEW

File: e2e/test/helpers/templates/FaultInjection.cs [hunk 1]
```diff
@@ -243,15 +213,6 @@ public static bool FaultShouldDisconnect(string faultType)
...
                 deviceClient.Dispose();
                 await testDevice.RemoveDeviceAsync().ConfigureAwait(false);
-
-                faultInjectionDuration.Stop();
-
-                TimeSpan timeToFinishFaultInjection = duration.Subtract(faultInjectionDuration.Elapsed);
-                if (timeToFinishFaultInjection > TimeSpan.Zero)
-                {
-                    logger.Trace($"{nameof(FaultInjection)}: Waiting {timeToFinishFaultInjection} to ensure that FaultInjection duration passed.");
-                    await Task.Delay(timeToFinishFaultInjection).ConfigureAwait(false);
```
Comment (abhipsaMisra) [inline e2e/test/helpers/templates/FaultInjection.cs:253]: the hub-wide faults might still need this delay.

Comment (drwill-ms) [reply e2e/test/helpers/templates/FaultInjection.cs:253]: Which are hub wide?

Comment (abhipsaMisra) [reply e2e/test/helpers/templates/FaultInjection.cs:253]: iirc that would be quota exceeded, throttled and bad auth.

**Pull request merged**